<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixWidget</class>
 <widget class="QWidget" name="MatrixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatrixWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Columns:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="width_spinbox">
       <property name="minimum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rows:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="height_spinbox">
       <property name="minimum">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>D- Matrix:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="d_minus_matrix">
       <property name="rowCount">
        <number>2</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <row/>
       <row/>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>D+ Matrix:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="d_plus_matrix">
       <property name="rowCount">
        <number>2</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <row/>
       <row/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>width_spinbox</zorder>
  <zorder>label_2</zorder>
  <zorder>height_spinbox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>width_spinbox</tabstop>
  <tabstop>d_plus_matrix</tabstop>
  <tabstop>d_minus_matrix</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
